apps/docs/app/globals.css

```diff
@@ -28,8 +33,11 @@ a {
 }
 
 .hd-wrapper {
-    width: min(100% - 3rem, 75rem);
```

Comment:
Why have you changed your approach here?

Reply:
While debugging an issue I wrongfully thought it was part of the issue, upon further inspection I can roll back to this approach.